RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,153 @@
+# Release Checklist
+
+This is a list of the things that need to happen during a release.
+
+## Build a Release
+
+### Prepare the Changelog (Full release only)
+
+1. Open the associated milestone. All issues and PRs should be closed. If
+   they are not you should reassign all open issues and PRs to future
+   milestones.
+1. Go through the commit history since the last release. Ensure that all PRs
+   that have landed are marked with the milestone. You can use this to
+   show all the PRs that are merged on or after YYY-MM-DD:
+   `https://github.com/issues?utf8=%E2%9C%93&q=repo%3Aapollographql%2Fapollo-cli+merged%3A%3E%3DYYYY-MM-DD`
+1. Go through the closed PRs in the milestone. Each should have a changelog
+   label indicating if the change is documentation, feature, fix, or maintenance. If
+   there is a missing label, please add one. If it is a breaking change, also add a changelog - BREAKING label.
+1. Choose an emoji for the release. Try to make it semi-related to something that's been included in the release.
+1. Add this release to the `CHANGELOG.md`. Use the structure of previous
+   entries. If you are releasing a beta or a release candidate, no official changelog is needed, but you're not off the hook! You'll need to write testing instructions in lieu of an official changelog.
```

Comment:
I feel like this point could be at the top or maybe next to this list since it’s not technically about the change log right?

Reply:
good point - addressed!